Fix formatting, analysis issues and null-safe issues

galunecandrey opened the pull request.

Renesanse commented:
Hello, @galunecandrey thx for your pull request, what kind of issues you solved in this pr?

galunecandrey commented:
> Hello, @galunecandrey thx for your pull request, what kind of issues you solved in this pr?
Hello! 
I fixed analyzer errors:

![Screenshot 2021-04-19 at 11 41 50](https://user-images.githubusercontent.com/16835611/115207801-96fc7280-a104-11eb-914d-eea694e782ff.png)

and 

![Screenshot 2021-04-19 at 11 42 12](https://user-images.githubusercontent.com/16835611/115208006-cdd28880-a104-11eb-8736-fc8675bce5a4.png)

Also this dependency was deprecated and need use `package:collection/collection.dart`:

![Screenshot 2021-04-19 at 11 43 24](https://user-images.githubusercontent.com/16835611/115208177-f78baf80-a104-11eb-8ecf-dc83ca451f78.png)

Also I fixed analyzer warnings and formatter issues

I decided to help because I saw it on `pub.dev`:

![Screenshot 2021-04-19 at 11 50 09](https://user-images.githubusercontent.com/16835611/115208595-679a3580-a105-11eb-98ef-e5e77c54fba6.png)

Renesanse commented:
ok, great, thx!
